AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4029,13 +4077,17 @@ function Debug-EntraKerbAdminConsent {
             Write-Verbose "CheckAdminConsent - START"
             $Context = Get-AzContext
             $TenantId = $Context.Tenant
+
+            # Detect if the Microsoft.Graph.Applications module is installed and at least version 2.2.0.
+            # Get-MgServicePrincipalByAppId was added in Microsoft.Graph.Applications v2.2.0.
+            Request-MSGraphModuleVersion -MinimumVersion 2.2.0
             
             Request-ConnectMsGraph `
                 -Scopes "DelegatedPermissionGrant.Read.All" `
                 -RequiredModules @("Microsoft.Graph.Applications", "Microsoft.Graph.Identity.SignIns") `
                 -TenantId $TenantId
 
-            Import-Module Microsoft.Graph.Applications
+            Import-Module Microsoft.Graph.Applications -MinimumVersion 2.2.0 -ErrorAction SilentlyContinue

```

Comment:
why do we continue if the module did not import?

Reply:
If the module is already imported, this call fails. So `SilentlyContinue` is needed to ignore that case.

As a side effect, this also makes it ignore any errors if it couldn't be imported. But in theory it shouldn't happen since we just ensured 2.2.0 is installed above. Regardless, even if it does fail, we'll just get an error later about the MS Graph module not being found when we try to use it, and the check fails -- so same result.